mix.exs

```diff
@@ -13,7 +13,7 @@ defmodule OMG.Umbrella.MixProject do
         "coveralls.detail": :test,
         "coveralls.post": :test,
         "coveralls.html": :test,
-        dialyzer: :prod
+        dialyzer: :test
```

Comment:
is this what we want? I recall setting this to `:prod` explicitly to omit dialyzing of test code and `test/support/` code, which is not built in `:prod` env.

(I know it caused a warning, but that was a specific warning, which perhaps could've been solved differently).

Hmmm, if I'm parsing github correctly, it seems like it passes the dialyzer step, then maybe it's an added benefit, that we cover test code with dialyzing? But anyway, I'm interested in the reasons/rationale here.

**EDIT** oh I see what you did [here](https://github.com/omisego/elixir-omg/pull/473/files#diff-9f06470aea4fd680aa740eedc568f669R5). Hm....

Reply:
I've always tried being thorough in using dialyzer so a future step would be to clean all dialyzer warnings from tests (because it also detects _unnecessary tests_)

Reply:
ok, let's try that, but let's remember this point to be able to back-track in case we reconsider.

I recall that dialyzing test/support code to be a huge pain once, but maybe we've given up to early.